python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -607,6 +609,10 @@ def test_array_function_series_fallback(series):
     tm.assert_equal(expect, got)
 
 
+@pytest.mark.skipif(
```

Comment:
These should be xfail and not skipif, imo.

Reply:
Done